src/test/run-make/raw-dylib-inline-cross-dylib/Makefile

```diff
@@ -0,0 +1,25 @@
+# Regression test for calling an inline function that uses a raw-dylib function.
+
+# only-windows
+
+include ../../run-make-fulldeps/tools.mk
+
+all:
+	$(RUSTC) --crate-type dylib --crate-name raw_dylib_test lib.rs -C prefer-dynamic
```

Comment:
It would be great to make sure that `raw_dylib_test.dll` does not export `inline_library_function` so that we know that we are actually testing what we think we are testing. Other run-make tests use `llvm-nm` or `llvm-objcopy` to take a peek at symbols -- but these tools don't seem to support DLLs. `dumpbin.exe /EXPORTS` works for me, but I can't quite tell if `dumpbin.exe` can be found in `PATH` reliably or if that's just on my system.

Reply:
> Other run-make tests use llvm-nm or llvm-objcopy to take a peek at symbols -- but these tools don't seem to support DLLs.

It works for `windows-gnu` DLLs probably because it puts symbol table into DLLs.

> I can't quite tell if dumpbin.exe can be found in PATH reliably or if that's just on my system.

It's part of MSVC toolchain so I guess it could be fine for tests that run only on `windows-msvc`.
Some of the ways that I know of for dumping imports table with LLVM are: `llvm-readobj --coff-exports <path>` and `llvm-objdump -p`.

Reply:
Done, thanks @mati865 - `llvm-objdump -p` worked for both GNU and MSVC.

Reply:
Great find, @mati865! Thanks!